expressjs/db/common/service.js

```diff
@@ -25,6 +45,7 @@ class BaseService {
   }
 
   deleteAllRows() {
+    // console.log('Delete all rows: ', this._name)
```

Comment:
Maybe want to remove this (or uncomment if this intended for logging and not debugging purposes) before merging?